rosette/src/misc.cc

```diff
@@ -134,13 +134,18 @@ properPrep (char* s)
 }
 #ifdef __GNUC__
 extern "C" {
-extern const char* const sys_errlist[];
-extern int sys_nerr;
+#include <string.h>
+//extern const char* const sys_errlist[];
```

Comment:
Remove cruft.  If you need the old code at some point, git will remember it for you.